src/svelte/App.svelte

```diff
@@ -1,7 +1,37 @@
 <script>
+  import AddMacroButton from "./Components/AddMacroButton.svelte";
+  import Macro from "./Components/Macro.svelte";
+  import Modals from "./Modals.svelte";
+
+  let macros = [];
```

Comment:
`macros `& `addMacro` in einem Store umsetzen?

Reply:
Können wir uns morgen vormittag anschauen dann :)

Reply:
done